File: parte001/ex005_cuadrado_cubo.java
// Elabore un algoritmo que lea un número y obtenga su cuadrado y su cubo.

import java.util.Scanner;

public class ex005_cuadrado_cubo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a;
        
        System.out.print("Ingrese el valor de A: ");
        a = sc.nextInt();
        
        System.out.println("A^2 = " + (a * a));
        System.out.println("A^3 = " + (a * a * a));

        sc.close();
    }
}

File: parte001/ex005_cuadrado_cubo.cpp
// Elabore un algoritmo que lea un número y obtenga su cuadrado y su cubo.

#include <iostream>

using namespace std;

int main()
{
    int a;
    cout << "Ingrese un valor entero: ";
    cin >> a;
    
    cout << "El cuadrado de " << a << " es " << a * a << endl;
    cout << "El cubo de " << a << " es " << a * a * a << endl;
    
    return 0;
}
